A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/Svg/A32NX_SvgConstraintElement.js

```diff
@@ -0,0 +1,68 @@
+class A32NXSvgConstraintElement extends SvgConstraintElement {
```

Comment:
Should we really extend the SvgConstraintElement? IMO it would be better just copying the original asobo file, renaming it, and then do your changes in there, especially since we are trying to hard fork, where we would need to copy this file anyways. @Benjozork What do you think?

Reply:
Well, I went in that direction first, but thought maybe extending is better since Asobo can add more things to the original class. But if we're talking a hard fork then It makes sense to just copy paste and add what we need.

Reply:
I think it would be preferable in this case to copy over the original file.

Reply:
Done